mapDownload.properties

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
The SF seems to work fine but when I switch to github then it just gives an error when I got to download maps.